_topic_map.yml

```diff
@@ -769,6 +769,8 @@ Topics:
   File: audit-log-policy-config
 - Name: Configuring TLS security profiles
   File: tls-security-profiles
+- Name: Seccomp Profile
```

Comment:
```suggestion
- Name: Seccomp Profiles
```
or how about
```suggestion
- Name: Seccomp
```

Reply:
@saschagrunert - Or can I call it Secure Computing or Secure Computing mode since seccomp is an acronym?

Reply:
Hm, seccomp is well known, so how about "Secure Computing / seccomp"?

Reply:
'/' is not acceptable in topic names as per doc standards. Are you ok with _Using seccomp profiles_?

Reply:
Yeah, SGTM